inventory/group_vars/all

```diff
@@ -1,5 +1,21 @@
+apache_ssl_use_stapling: yes
+
 hoist_repo: https://github.com/BonnyCI/hoist.git
 
+letsencrypt_account_key_content: "{{ secrets.letsencrypt.account_key }}"
+letsencrypt_csr_dn:
+  C: AU
+  ST: NSW
+  L: Australia
+  O: BonnyCI
+letsencrypt_configure_redirect: yes
+letsencrypt_email: jamielennox@gmail.com
+
+nodepool_mysql_host: zuul.bonnyci-internal.portbleu.com
```

Comment:
rattboi: so by break i think you are referring to nodepool_mysql_host in all. I don't know why i moved this as part of this review, i'm guessing it was a mistake. However no, the order vars files are loaded in will mean that this will have the lowest priority and be overriden by everything else so it won't actually break anything - but it still shouldn't be there.